src/app/widgets/reactive-fields/reactive-label/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './reactive-label.module';
+export * from './reactive-label.component';
```

Comment:
This line doesn't need here. We jast import the module that exports this component